modules/ixBidAdapter.js

```diff
@@ -66,15 +102,27 @@ function parseBid(rawBid, currency) {
   }
 
   bid.requestId = rawBid.impid;
-  bid.width = rawBid.w;
-  bid.height = rawBid.h;
-  bid.ad = rawBid.adm;
+
   bid.dealId = utils.deepAccess(rawBid, 'ext.dealid');
-  bid.ttl = TIME_TO_LIVE;
   bid.netRevenue = NET_REVENUE;
   bid.currency = currency;
   bid.creativeId = rawBid.hasOwnProperty('crid') ? rawBid.crid : '-';
 
+  // in the event of a video
+  if (utils.deepAccess(rawBid, 'ext.vasturl')) {
+    bid.vastUrl = rawBid.ext.vasturl
```

Comment:
Ok so you are setting the `vastUrl`.

But no `videoCacheKey`.

What this means that if there is a `cache.url` configured then it will attempt to cache your bid once again using client side caching.

This would result in your bid response to look like so:

![image](https://user-images.githubusercontent.com/13972794/69113281-6dcf4280-0a37-11ea-9760-3267d61ca8fa.png)

Where the `hb_cache_id` and `hb_uuid` are both the client side keys returned from the cache (I used Rubicons cache in this example.

Reply:
The reason I bring this up is because it all depends on how a publisher utilized Prebid Video auctions.

A lot of times it may not be a standard creative

It may use the DFP VAST creatives which need to point to a specific Vast url with a pattern match macro.

So this scenario might be a little tricky.

If the publisher has a custom setup where they just are parsing the bid responses and adding them to the VAST player, then this should probably be fine.

Reply:
I just wanted to make you aware of this case and make sure you expect it or are okay with it.

I plan to merge this PR before the new release tomorrow (Tuesday).

But will give you until then to make a change or ask questions.

@ix-prebid-support

Reply:
Hi Robert - 

We're OK with it for now. Thank you for pointing it out. 

Regards,
Dmitry
Index Exchange